src/app/mod.rs

```diff
@@ -474,7 +471,7 @@ where
                     self.broker
                         .consume(
                             queue,
-                            Box::new(move || {
+                            Box::new(move |_| {
```

Comment:
```suggestion
                            Box::new(move |e| {
```

TODO: at least log this error. Or better yet, send it through the `broker_error_tx`.